<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryDialog</class>
 <widget class="QDialog" name="GeometryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>22</y>
     <width>764</width>
     <height>293</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Geometry Setting</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>horizontal size</string>
            </property>
            <property name="buddy">
             <cstring>ln_horizontalSize</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ln_horizontalSize">
            <property name="text">
             <string>-10 10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>vertical size</string>
            </property>
            <property name="buddy">
             <cstring>ln_verticalSize</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ln_verticalSize">
            <property name="text">
             <string>-50 0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>start point</string>
            </property>
            <property name="buddy">
             <cstring>ln_startPoint</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ln_startPoint">
            <property name="text">
             <string>15 0 0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_points">
          <property name="text">
           <string>points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lbl_spread">
          <property name="text">
           <string>Spread</string>
          </property>
          <property name="buddy">
           <cstring>txtEdit_spread</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtEdit_spread"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="btn_setSource">
        <property name="text">
         <string>set source</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_OK">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
